### ISSUE TITLE
Missing config shows old/wrong url

### ISSUE THREAD

**Issue by rduivenvoorde** [MapServer/MapServer#6778]
When something goes wrong with the new config file, you see (both in terminal and in html):

```
 msLoadConfig(): Unable to access file. See mapserver.org/config_file.html for more information. 
```

but https://mapserver.org/config_file.html gives a not found...

Either there is a redirection missing on web server level, OR the url should contain something like (?)  https://mapserver.org/mapfile/config.html

Not sure what is preferred?

**Comment by jmckenna** [comment]
Hmm I thought I had fixed that earlier through https://github.com/MapServer/MapServer/pull/6625 

@rduivenvoorde I think this was fixed post-8.0 release, maybe you are trying with that version?  (I wonder if you could try with `main` too.  Let me know.  thanks.

**Comment by rduivenvoorde** [comment]
@jmckenna yeah, sorry I did not show version info. I'm on Debian Testing which has 8.0.0 

Thanks, sorry for the fuzz...
